optimum/habana/transformers/generation/utils.py

```diff
@@ -974,14 +979,16 @@ def generate(
         # 1. Handle `generation_config` and kwargs that might update it, and validate the `.generate()` call
         self._validate_model_class()
         tokenizer = kwargs.pop("tokenizer", None)  # Pull this out first, we only use it for stopping criteria
+        #assistant_tokenizer = kwargs.pop("assistant_tokenizer", None)  # only used for assisted generation
```

Comment:
please remove all this commented out codes.

Reply:
I think it is also affected by transformers 4.47. Will remove or uncomment it when transformers has its v4.47.0 release & optimum habana update to match 4.47!

Reply:
Removed.